adafruit_character_lcd/character_lcd.py

```diff
@@ -693,15 +708,15 @@ def color(self):
         return self._color
 
     @color.setter
-    def color(self, color):
+    def color(self, color: Union[List[int], int]) -> None:
         if isinstance(color, int):
             if color >> 24:
                 raise ValueError("Integer color value must be positive and 24 bits max")
             # NOTE: convert to 0-100
-            r = (color >> 16) / 2.55
-            g = ((color >> 8) & 0xFF) / 2.55
-            b = (color & 0xFF) / 2.55
-            color = [r, g, b]
+            r = int((color >> 16) / 2.55)
+            g = int(((color >> 8) & 0xFF) / 2.55)
+            b = int((color & 0xFF) / 2.55)
+            color = [r, g, b]  # This is List[float], should be List[int]
```

Comment:
I see the use case you were mentioning.  I don't think it's actually an issue since the output of `_map()` below gets wrapped by `int()`, so no need to wrap `r`, `g`, or `b` with `int()` beforehand - it's previously implementation was okay since `_map()` can take `float`.

Reply:
OK, thx.

Reply:
Also, should the typing for the function is `def color(self, color: Union[List[int], int]) -> None:` . Should it be `def color(self, color: Union[List[float], int]) -> None:` ?

Reply:
That's correct, it looks like upon further inspection is asking for a percentage of 0% to 100%.  Yeah, let's use `List[float]`, good catch!

Reply:
I've gone with Union[List[float, float, float], int]

Reply:
When using `List`, you only specify the type of any element, not how many elements there are, so `Union[Lits[float], int]` is correct.